<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Recursos/Iconos/pintar.png</normaloff>:/Recursos/Iconos/pintar.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuEstilo">
    <property name="title">
     <string>Estilo</string>
    </property>
    <addaction name="actionColor"/>
    <addaction name="actionAncho"/>
   </widget>
   <widget class="QMenu" name="menuForma">
    <property name="title">
     <string>Forma</string>
    </property>
    <addaction name="actionLineas"/>
    <addaction name="actionLibre"/>
    <addaction name="actionRectangulos"/>
    <addaction name="actionCircunferencias"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEstilo"/>
   <addaction name="menuForma"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbrir"/>
   <addaction name="actionNuevo"/>
   <addaction name="actionGuardar"/>
   <addaction name="actionSalir"/>
   <addaction name="separator"/>
   <addaction name="actionColor"/>
   <addaction name="actionAncho"/>
   <addaction name="separator"/>
   <addaction name="actionLineas"/>
   <addaction name="actionLibre"/>
   <addaction name="actionCircunferencias"/>
   <addaction name="actionRectangulos"/>
  </widget>
  <action name="actionNuevo">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Recursos/Iconos/agregar.png</normaloff>:/Recursos/Iconos/agregar.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Recursos/Iconos/salvar.png</normaloff>:/Recursos/Iconos/salvar.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Recursos/Iconos/papelera-de-reciclaje.png</normaloff>:/Recursos/Iconos/papelera-de-reciclaje.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Recursos/Iconos/lapices-de-color.png</normaloff>:/Recursos/Iconos/lapices-de-color.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionAncho">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Recursos/Iconos/conjunto-cuadrado.png</normaloff>:/Recursos/Iconos/conjunto-cuadrado.png</iconset>
   </property>
   <property name="text">
    <string>Ancho</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionLineas">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Recursos/Iconos/lapiz.png</normaloff>:/Recursos/Iconos/lapiz.png</iconset>
   </property>
   <property name="text">
    <string>Lineas</string>
   </property>
  </action>
  <action name="actionLibre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Recursos/Iconos/pincel-de-arte.png</normaloff>:/Recursos/Iconos/pincel-de-arte.png</iconset>
   </property>
   <property name="text">
    <string>Libre</string>
   </property>
  </action>
  <action name="actionRectangulos">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Recursos/Iconos/rectangulo.png</normaloff>:/Recursos/Iconos/rectangulo.png</iconset>
   </property>
   <property name="text">
    <string>Rectánculos</string>
   </property>
   <property name="toolTip">
    <string>Rectángulos</string>
   </property>
  </action>
  <action name="actionCircunferencias">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Recursos/Iconos/brujula.png</normaloff>:/Recursos/Iconos/brujula.png</iconset>
   </property>
   <property name="text">
    <string>Circunferencias</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Recursos/Iconos/abrir-documento.png</normaloff>:/Recursos/Iconos/abrir-documento.png</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
